lib/erlef/erlef/blog/post.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Erlef.Blog.Post do
+  @moduledoc """
+  Schema for blog posts.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "blog_posts" do
+    field(:body, :string)
+    field(:slug, :string)
+    field(:status, :string)
+    field(:title, :string)
+    field(:member_id, :id)
+    field(:working_group_id, :id)
+
+    many_to_many(:blog_tags, Erlef.Blog.Tag, join_through: "blog_posts_blog_tags")
+    embeds_many(:versions, Erlef.Blog.Version, on_replace: :delete)
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(post, attrs) do
+    post
+    |> cast(attrs, [:title, :slug, :body, :status])
+    |> validate_required([:title, :slug, :body, :status])
```

Comment:
Do you think `member_id` should be required?

Reply:
Yes, definitely! Thanks for spotting that.